src/components/ContainerCode.tsx

```diff
@@ -33,7 +33,12 @@ export const ContainerCode: React.FC<ContainerCodeProps> = ({
   pauseOnFocusLoss,
   isDefaultProps,
   draggable,
-}) => (
+  theme = "light",
+}) => {
+  console.log({theme});
```

Comment:
Hey sorry for the late reply. You can drop the console.log

Reply:
Oh, no problem! Sorry I mine was late too  :) 

I've removed the the console.log. You can check out.
Once again, thanks for the work. Your package is really great!